### ISSUE TITLE
Location of photo of the day

### ISSUE THREAD

**Issue by si-the-pie** [cyclestreets/mobileweb2022#1]
# Photo
![BF5CB22A-9613-4C26-B41A-1264ED822DEC](https://user-images.githubusercontent.com/361987/103074914-77c05f00-45c2-11eb-9374-350c8b4cf7ba.png)
Scene on following this photo of the day link:
https://twitter.com/cyclestreets/status/1342017166267314178?s=21

# Map
![3A81C72D-027F-4D23-99C2-815EE4081B42](https://user-images.githubusercontent.com/361987/103075415-7479a300-45c3-11eb-92e9-7ca343c91b9f.png)
Closing the overlay shows map around current location, not County Down where the photo is located .

# Where is it?
One reason users follow Photo of the day from Twitter, or follow a photo map location link more generally is to determine where the image is located.

# Suggestion
Turn off “use current location mode” when following location link, allowing user to close overlay and explore area of photo.

**Comment by mvl22** [comment]
Yes, this is missing a map centring upon popup, with a disabling of the current location just before that.

This will require a fix in the `apiCallId` handling part of the layerviewer library, which we trigger as a result of this declaration:
https://github.com/cyclestreets/mobile-design/blob/master/js/cyclestreets.js#L315-L324

**Comment by mvl22** [comment]
The fix is basically to add at
https://github.com/cyclestreets/Mapboxgljs.LayerViewer/blob/master/src/layerviewer.js#L2342

- Disable geolocation
- Move map to the lat/lon in features[0]

**Comment by si-the-pie** [comment]
It occurs to me that during the `add photo` procedure using the mobileweb one of the screens shows the location of the photo alreay - when prompting to add details. In the screenshot I've scrolled the marker into view. So could this screen be re-purposed in some way?


![IMG_FE8BF40FF7F8-1](https://user-images.githubusercontent.com/361987/106804473-5640a180-665d-11eb-961c-cfd552420d71.jpeg)

https://cambridge.cyclestreets.net/location/162426/

**Comment by mvl22** [comment]
No, that would need a lot of changes. Don’t forget the flip side of the pop up too.

Digging into that UI would still require the same fix, which is identified above.
